config/secrets.js

```diff
@@ -59,3 +57,17 @@ module.exports.updateFetching = function(flag, cb) {
     return cb(err);
   });
 };
+
+// update settings
+module.exports.update = function(type, settings, cb) {
+  cb = cb || function() {};
+
+  for (var key in settings) {
+    var item = type + ':' + key;
+    nconf.set(item, settings[key]);
+  }
+
+  nconf.save(function(err) {
+    return cb(err);
+  });
+};
```

Comment:
So I take it that this is temporary code, and will go away once the settings are moved to the DB?